### PR TITLE
2004 - Closing GNSS station

### PR DESCRIPTION
**Pull request by aleks-spes**
Closing GNSS station - 2004, after it has been replaced with PKTP

### CODE REVIEW

**Comment by elidana** [comment]
hi @aleks-spes , is this one ready to be reviewed?

File: network/monuments.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Mark,Domes Number,Mark Type,Type,Ground Relationship,Foundation Type,Foundation Depth,Start Date,End Date,Bedrock,Geology
-2004,,Forced Centering,Steel Mast,0,Reinforced Concrete,0,2000-06-19T00:00:00Z,9999-01-01T00:00:00Z,,
+2004,,Forced Centering,Steel Mast,0,Reinforced Concrete,0,2000-06-19T00:00:00Z,2024-07-05T00:00:00Z,,
```
Comment (elidana) [inline network/monuments.csv:2]: has the monument been removed from the site? If not, the end date here can remain "open"

Comment (aleks-spes) [reply network/monuments.csv:2]: As far as I know the monument has been removed (from one of the comments in the ticket: `2004 mark is still within the fenced area if it ever needs to be utilised, but the monument has been returned to GNS`)
The thing I am concerned is if we should close site in the `marks.csv` table or not. The geodetic mark is still there, but I don't think we will occupy it again. What do you think @elidana ?

Comment (elidana) [reply network/monuments.csv:2]: oh no, if there was some form of antenna mount installed over the geodetic mark on the ground then yes, agree with you that the "monumnet' can be considered as closed as well! Sorry for the confusion!

Comment (aleks-spes) [reply network/monuments.csv:2]: No worries! This is exactly what I wanted to discuss. Will push the PR to the review.

**Comment by aleks-spes** [comment]
Hey @elidana @EvanCGNS @CBurton90 @gva6500 !
This PR is ready for review. Could you please review it?

**Review by elidana — CHANGES_REQUESTED** [review]
this looks good (also considering equipment closing timestamps added in https://github.com/GeoNet/delta/pull/2136) and last epoch recorded at `2004` which is
`> 2024  7  4  0 23  0.0000000  0 19       0.000000002000`


For completeness, can you please also add closing timestamp to the `environment/visibility.csv` record? After that we can approve/merge this one. 
Thanks!

**Review by EvanCGNS — DISMISSED** [review]
LGTM! As there's nothing in environment/visibility.csv for 2004 (temporary site), I'd be happy to merge as is. Otherwise, feel free to create a line for completeness.

**Comment by aleks-spes** [comment]
Hey @elidana! Unfortunately, we don't have a record for `2004` in the `visibility.csv`. Most likely I forgot to add it at the site opening. Do you think we need to add this record now?

**Comment by elidana** [comment]
aah ok! yes, I think might be good to add a record for consistency. Thanks!

**Review by elidana — APPROVED** [review]
looks good, thanks @aleks-spes !

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**